lib/oli_web/live/common/hierarchy/hierarchy_picker.ex

```diff
@@ -121,15 +124,34 @@ defmodule OliWeb.Common.Hierarchy.HierarchyPicker do
         <%= render_breadcrumb assigns %>
       </div>
       <div class="hierarchy">
-        <%= for pub <- @publications do %>
+        <form phx-debounce="500" phx-change="HierarchyPicker.publications_text_search" class="ml-auto w-56">
+          <SearchInput.render text={assigns.publications_table_model_params[:text_search]} name="text_search" id="text_search" placeholder="Search publications" />
+        </form>
+        <PagedTable.render
+          __context__={assigns[:__context_]}
+          total_count={assigns.publications_table_model_total_count}
+          filter={assigns.publications_table_model_params.text_filter}
+          limit={assigns.publications_table_model_params.limit}
+          offset={assigns.publications_table_model_params.offset}
+          table_model={assigns.publications_table_model}
+          allow_selection={false}
+          sort=""
+          page_change="HierarchyPicker.publications_page_change"
+          selection_change=""
+          show_top_paging={false}
+          show_bottom_paging={true}
+          additional_table_class="remix_materials_publications_table"
+          render_top_info={false}
+        />
+        <%!-- <%= for pub <- @publications do %>
 
           <div id={"hierarchy_item_#{pub.id}"}>
             <button class="btn btn-link ml-1 mr-1 entry-title" phx-click="HierarchyPicker.select_publication" phx-value-id={pub.id}>
               <%= pub.project.title %>
             </button>
           </div>
 
-        <% end %>
+        <% end %> --%>
```

Comment:
Is this `--%>` leftover from an old comment block?

Reply:
Oops